src/auth/auth.controller.ts

```diff
@@ -72,19 +72,25 @@ export class AuthController {
       );
     }
 
-    const authStatus = await this.authService.verifyUserFromAuthCode({
+    const authDetails = await this.authService.verifyUserFromAuthCode({
       code,
       nonce: cookieInstance.nonce,
       codeVerifier: cookieInstance.codeVerifier,
     });
 
     // TODO - Have the bot send different messages depending on outcome
-    if (authStatus === SgidAuthStatus.AUTHENTICATED_PUBLIC_OFFICER) {
+    if (authDetails.status === SgidAuthStatus.AUTHENTICATED_PUBLIC_OFFICER) {
+      const verifiedMessage = authDetails.poDetails.map((object) => {
+        return `You are verified with the following details: 
+        \nAgency: ${object.agency_name}
+        \nDepartment: ${object.department_name}
+        \nTitle: ${object.employment_title}`;
```

Comment:
nit - since you are using backticks, the line breaks you have used to make the block of text are honoured. You can hence do...
```suggestion
        Agency: ${object.agency_name}
        Department: ${object.department_name}
        Title: ${object.employment_title}`;
```